backend/yonder/views.py

```diff
@@ -301,11 +301,25 @@ def retrieve(self, request, author_id, follower_id):
         return Response(status=status.HTTP_404_NOT_FOUND)
 
     def destroy(self, request, author_id, follower_id):
-        author_follower = get_object_or_404(AuthorFollower, author=author_id, follower__id=follower_id)
-        author_follower.delete()
+        try:
+            author_follower = AuthorFollower.objects.get(author=author_id, follower__id=follower_id)
+            author_follower.delete()
+            return Response(status=status.HTTP_204_NO_CONTENT)
+            
+        except AuthorFollower.DoesNotExist:
 
-        return Response(status=status.HTTP_204_NO_CONTENT)
+            authorToUnfollow = request.data["actor"]
```

Comment:
`actor` is the person requesting the unfollow, so `authorToUnfollow` should be `request.data["object"]`